examples/flutter_gallery/lib/demo/cupertino/cupertino_segmented_control_demo.dart

```diff
@@ -53,7 +55,10 @@ class _CupertinoSegmentedControlDemoState extends State<CupertinoSegmentedContro
     return Scaffold(
       appBar: AppBar(
         title: const Text('Segmented Control'),
-        actions: <Widget>[MaterialDemoDocumentationButton(CupertinoSegmentedControlDemo.routeName)],
+        actions: <Widget>[
+          MaterialDemoDocumentationButton(
+              CupertinoSegmentedControlDemo.routeName)
+        ],
```

Comment:
reading this i can't immediately tell what's going on, it takes more effort to figure out what the nesting than the original.

Reply:
This case and similar look reasonable with a line length of 110.